src/model/guild/guild_id.rs

```diff
@@ -568,6 +568,32 @@ impl GuildId {
         http.as_ref().edit_member(self.0, user_id.0, &map)
     }
 
+    /// Disconnects a member from a voice channel in a guild.
+    ///
+    /// Requires the [Move Members] permission.
+    ///
+    /// [Move Members]: ../permissions/struct.Permissions.html#associatedconstant.MOVE_MEMBERS
+    #[cfg(feature = "http")]
```

Comment:
This feature gate is pointless. The `model` feature depends on the `http` feature, so there is no need to check the latter.
```suggestion

```